testing/features/components/navigation.feature

```diff
@@ -1,3 +1,4 @@
+# NB: Due to a Storybook issue, these tests won't run on Old Safari / iOS
```

Comment:
What's the actual issue? Longer term we could probably start running these tests against the `demo/` app which is a much simpler and more realistic environment than storybook but we should chat about that first 🤓

Reply:
Go for it. Happy to let you guys decide on this. You probably know better.

P.S. Showing my age but: https://www.youtube.com/watch?v=OkGaq9xiQZY

Reply:
Hah, I see. Didn't make that connection.

Yeah, that probably makes sense it'd be some compatibility thing. Storybook doesn't have a very good cross-browser story. No need to switch now, but as we've migrate more components into the Rails engine there will be a pretty decent test environment within the demo app. Something to come back to.

Reply:
I've popped a new Phase I ticket in the backlog for this. Happy to tackle this whenever you feel is appropriate. I don't envisage it being a lot of work our end to migrate the tests (Probably just altering some page urls / ruby setup code).